core/src/main/java/hudson/model/UpdateCenter.java

```diff
@@ -1177,7 +1177,7 @@ public File download(DownloadJob job, URL src) throws IOException {
                 File dst = job.getDestination();
                 File tmp = new File(dst.getPath()+".tmp");
 
-                LOGGER.info("Downloading "+job.getName());
+                LOGGER.finest("Downloading "+job.getName());
```

Comment:
We need messages related to plugin installation, but I think we have some redundancy there. Could go either way on this one.

Reply:
I would keep the original level. Plugin installations are pretty rare, and they are related to the Jenkins system health

Reply:
```suggestion
                LOGGER.info("Downloading "+job.getName());
```